contrib/swtpm/template.py

```diff
@@ -0,0 +1,58 @@
+pkgname = "swtpm"
+pkgver = "0.8.1"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--with-gnutls"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "bash",
+    "expect",
+    "gawk",
+    "gmake",
+    "gnutls-progs",
+    "libtool",
+    "pkgconf",
+    "socat",
+]
+makedepends = [
+    "gnutls-devel",
+    "json-glib-devel",
+    "libseccomp-devel",
+    "libtasn1-devel",
+    "libtpms-devel",
+    "linux-headers",
+    "openssl-devel",
+]
+checkdepends = [
+    "iproute2",
+    "trousers",
+]
+depends = ["gnutls-progs"]
+pkgdesc = "TPM emulator with socket, character device, and Linux CUSE interface"
+maintainer = "cesorious <cesorious@gmail.com>"
+license = "BSD-3-Clause"
+url = "https://github.com/stefanberger/swtpm"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "7bba52aa41090f75087034fac5fe8daed10c3e7e7234df7c9558849318927f41"
+
+
+def post_install(self):
+    self.install_license("LICENSE")
+
+
+@subpackage("libswtpm")
+def _libs(self):
+    return ["usr/lib/swtpm/*.so.*"]
+
+
+@subpackage("libswtpm-devel")
+def _devel(self):
+    return self.default_devel()
+
+
+@subpackage("libswtpm-devel-static")
```

Comment:
drop this one and instead the .a's in swtpm-libs (cbuild will split it automatically)

Reply:
it doesn't cause it's in /usr/lib/swtpm (i had the same thought)

Reply:
or do you mean it works when it ends up in -libs proper?

Reply:
er i meant -devel, not -libs

Reply:
the static splitter does an rglob on usr/lib, it should work in subdirs